openrouteservice/src/main/java/org/heigit/ors/fastisochrones/partitioning/PartitioningDataBuilder.java

```diff
@@ -0,0 +1,74 @@
+package org.heigit.ors.fastisochrones.partitioning;
+
+import com.graphhopper.routing.util.EdgeFilter;
+import com.graphhopper.storage.Graph;
+import com.graphhopper.util.EdgeExplorer;
+import com.graphhopper.util.EdgeIterator;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Creates the data necessary for running a max flow min cut algorithm.
+ *
+ * @author Hendrik Leuschner
+ */
+public class PartitioningDataBuilder {
+    private Graph graph;
+    private EdgeExplorer edgeExplorer;
+    private PartitioningData pData;
+    private EdgeFilter edgeFilter;
+    private int maxEdgeId = -1;
+
+    PartitioningDataBuilder(Graph graph, PartitioningData pData) {
+        this.graph = graph;
+        this.edgeExplorer = this.graph.createEdgeExplorer();
+        this.pData = pData;
+    }
+
+    public void run() {
+        //Need entries for all edges + one dummy edge for all nodes
+        pData.createEdgeDataStructures(graph.getAllEdges().length() + 1);
+        pData.fillFlowEdgeBaseNodes(graph);
+        pData.createNodeDataStructures(graph.getNodes() + 1);
+        buildStaticNetwork();
+    }
+
+    public void buildStaticNetwork() {
+        Set<Integer> targSet = new HashSet<>();
+        EdgeIterator edgeIter;
+
+        for (int baseId = 0; baseId < graph.getNodes(); baseId++) {
+            targSet.clear();
+            edgeIter = edgeExplorer.setBaseNode(baseId);
+            while (edgeIter.next()) {
+                int targId = edgeIter.getAdjNode();
+                if (!acceptForPartitioning(edgeIter))
+                    continue;
+                //>> eliminate Loops and MultiEdges
+                if ((baseId != targId) && (!targSet.contains(targId))) {
+                    targSet.add(targId);
+                    addEdge(edgeIter.getEdge(), baseId, targId);
```

Comment:
Probably I'm missing something here, but it looks as if `addEdge` would effectively just set all corresponding entries in `PartitionigData.flow` to `false` which seems obsolete to me as the whole array is by default initialized to false?

Reply:
No, I think you are right, good point. This is a remainder of code whose use vanished due to optimization, but I did not see that this makes the functionality completely unnecessary. Removed